db/migrate/20150127045938_create_replications.rb

```diff
@@ -0,0 +1,10 @@
+class CreateReplications < ActiveRecord::Migration
+  def change
```

Comment:
if you make replication a subclass of debit-transaction, you won't need a separate table for replications, just an extra column called "type" in the transactions table.